Yoast/wordpress-seo
# Social conditionals

## Summary


This PR can be summarized in the following changelog entry:

* 

## Relevant technical choices:

* Use the context object to determine whether or not OpenGraph is enabled.

## Test instructions

This PR can be tested by following these steps:

**To test whether Twitter only falls back to Open Graph when Open Graph is enabled**
* Create or edit a post. 
* In the post's metabox, click `Social` and add a Facebook title and description, and a Twitter title and description.
* In `SEO` --> `Social` --> `Twitter`, make sure that Twitter is enabled.
* In `SEO` --> `Social` --> `Facebook`, make sure that Open Graph is enabled.
* In the frontend, check whether `twitter:title` and `twitter:description` contain the values you just entered.
* In the backend, empty the `Twitter title` and `Twitter description` fields.
* In the frontend, check whether `twitter:title` and `twitter:description` contain the values you just entered for Facebook.
* In the backend, disable Open Graph.
* In the frontend, check whether `twitter:title` is equal to `%%title%% %%page%% %%sep%% %%sitename%%`
* In the frontend, check whether `twitter:description` is equal to an excerpt from the post.

**To test whether Twitter tags are not rendered when Twitter is disabled by a hook**
* In `SEO` --> `Social` --> `Twitter`, make sure that Twitter is enabled.
* In the frontend, inspect any page (for example the post you used above) and see that Twitter tags are rendered.
* In the code, add the `wpseo_output_twitter_card` and set it to false.
    * For example, paste the following code into the `functions.php` file of the theme you are using: `add_filter( 'wpseo_output_twitter_card', '__return_false' );`
* In the frontend, inspect any page (for example the post you used above) and see that Twitter tags are NOT rendered.

## UI changes

## Documentation

## Quality assurance


Fixes #
